### PR TITLE
feat (manager) : components & pages added

### PR DESCRIPTION
**Pull request by soumyadip007**
**Components Added:** 

- Chart Component :

1. Deployment Week
2. Total Deployment 

- Settings Component : 

1. API Key 
2. Delete Spa
3. Manage Spa

- Web-Property Component : 

1. Activity Stream
2. Spa Property
3. Web Property

- Utils:

1. API Request 

**Pages Added:**

1. Index Page (Route - /properties)
2. Web Property Page ( Route - /properties/{web-property-name} )
3. Spa Proeprty Page ( Route - /properties/{web-property-name}/spa/{spa-name} )
5. Web Property Settings Page ( Route - /properties/{web-property-name}/settings )

### CODE REVIEW

File: packages/manager/components/chart/aggregate-chart.tsx [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+  ChartAxis,
+  ChartGroup,
+  ChartLine,
+  ChartVoronoiContainer,
+} from "@patternfly/react-charts";
+import React, { FunctionComponent } from "react";
+import { AnyProps } from "../models/props";
+
+const AggregateChart: FunctionComponent<AnyProps> = ({ type, props }: AnyProps) => {
+  if (type === "donut")
```
Comment (arkaprovob) [inline packages/manager/components/chart/aggregate-chart.tsx:12]: if (type === "donut") It seems the check value is hard coded. Is it normal? @soumyadip007 @hybridx

Comment (hybridx) [reply packages/manager/components/chart/aggregate-chart.tsx:12]: We can create an object here

Comment (hybridx) [reply packages/manager/components/chart/aggregate-chart.tsx:12]: chartType.donut and chartType.month

Comment (soumyadip007) [reply packages/manager/components/chart/aggregate-chart.tsx]: @arkaprovob  please check now,

File: packages/manager/styles/globals.css [hunk 2]
```diff
@@ -71,3 +72,29 @@ button.spaship_btn {
 * {
   box-sizing: border-box;
 }
+
+.spaship-circle {
+  border-radius: 999px;
```
Comment (hybridx) [inline packages/manager/styles/globals.css:77]: @soumyadip007 why 999px? use 50% instead

**Review by arkaprovob — APPROVED** [review]

**Review by hybridx — CHANGES_REQUESTED** [review]

File: packages/manager/styles/globals.css [hunk 3]
```diff
@@ -71,3 +72,29 @@ button.spaship_btn {
...
+  width: 70px;
+  height: 70px;
+  margin-top: 25px;
+  margin-left: 25px;
+  background: hsl(42, 98%, 54%);
```
Comment (hybridx) [inline packages/manager/styles/globals.css:82]: @soumyadip007 can we create variables instead, would be easier to manage in the future

File: packages/manager/styles/globals.css (hunk 2 above)
Comment (soumyadip007) [reply packages/manager/styles/globals.css:77]: Yess done

File: packages/manager/styles/globals.css [hunk 4]
```diff
@@ -71,3 +72,29 @@ button.spaship_btn {
...
+  color: var(--spaship-global--Color--spaship-gray);
+  text-align: center;
+  transition: color 0.2s linear;
+  font: 70px arial, sans-serif;
```
Comment (hybridx) [inline packages/manager/styles/globals.css:86]: Font size here

**Review by hybridx — CHANGES_REQUESTED** [review]

File: packages/manager/styles/globals.css (hunk 4 above)
Comment (soumyadip007) [reply packages/manager/styles/globals.css]: Resolved

**Review by hybridx — APPROVED** [review]

**Pull request merged**